liquidity/autoloop_testcontext_test.go

```diff
@@ -266,29 +267,48 @@ func (c *autoloopTestCtx) autoloop(step *autoloopStep) {
 	// recommended swaps. Note that this differs from our set of expected
 	// swaps because we may get quotes for suggested swaps but then just
 	// log them.
+	var check bool
 	for _, expected := range step.quotesIn {
 		request := <-c.quoteRequestIn
-		assert.Equal(
-			c.t, expected.request.Amount, request.Amount,
-		)
 
-		assert.Equal(
-			c.t, expected.request.HtlcConfTarget,
-			request.HtlcConfTarget,
-		)
+		check = false
```

Comment:
Maybe we could cache the `request` received, order them and compare them against `step.quotesIn`?